templates/tftmpl/golden_test.go

```diff
@@ -69,6 +70,7 @@ func TestNewFiles(t *testing.T) {
 			Func:   newVariablesTF,
 			Golden: "testdata/variables.tf",
 			Input: RootModuleInputData{
+				TerraformVersion: goVersion.Must(goVersion.NewSemver("0.14.2")),
```

Comment:
My nose started twitching when I saw a version hardcoded in a string but it looks like it is more an example than something that actually uses that version. If I'm wrong and it is actually using that maybe it would be better moved to a constant. If I'm right maybe change it to something that would make it more obvious that it isn't real (like 9.99.9 or something).

Reply:
Not an example. This is a test case. It's no different than another test value that has hard coded strings. Yeah I can change it to a nonsensical version